### PR TITLE
fixed code snippet in p. 1.4 (Classes)

### PR DESCRIPTION
**Pull request by ivan-egorov42**
According to #6. 
@yegor256 @Graur  please, have a look

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -427,16 +427,22 @@ \subsection{Classes}
 It can be mapped to the following EO code, where a class becomes a factory of objects:
 
 \begin{ffcode}
++alias org.eolang.cage
++alias org.eolang.txt.sprintf
++alias org.eolang.io.stdout
++alias org.eolang.hamcrest.assert-that
+
```
Comment (Graur) [inline paper.tex]: @ivan-egorov42 I think it would be better to use `Home` form of aliases - `QQ`.

File: paper.tex [hunk 2]
```diff
@@ -427,16 +427,22 @@ \subsection{Classes}
 It can be mapped to the following EO code, where a class becomes a factory of objects:
 
 \begin{ffcode}
++alias org.eolang.cage
++alias org.eolang.txt.sprintf
++alias org.eolang.io.stdout
++alias org.eolang.hamcrest.assert-that
```
Comment (Graur) [inline paper.tex]: @ivan-egorov42 I don't see any `assert-that` objets in this code. Do really we need it here?

File: paper.tex [hunk 3]
```diff
@@ -427,16 +427,22 @@ \subsection{Classes}
...
   [i] > new
     cage 0 > id
     seq > @
       id.write i
       []
         [] > path
-          QQ.io.sprintf > @
-            "/tmp/%s.txt"
-            id
+          stdout > @
```
Comment (Graur) [inline paper.tex]: @ivan-egorov42 I think that we do not need this object here, since it was absent before

**Review by Graur — CHANGES_REQUESTED** [review]
@ivan-egorov42 Good one! Just a few comments below

File: paper.tex (hunk 3 above)
Comment (ivan-egorov42) [reply paper.tex]: @Graur I thought that `path` should print message to console. Otherwise what's the point in it?

File: paper.tex (hunk 2 above)
Comment (ivan-egorov42) [reply paper.tex]: My bad, I'll remove this.

File: paper.tex (hunk 1 above)
Comment (ivan-egorov42) [reply paper.tex]: Ok, I can change this.

File: paper.tex (hunk 3 above)
Comment (Graur) [reply paper.tex]: @ivan-egorov42 The point is to get the path. If you make such changes, fix also a Ruby code.

**Review by Graur — CHANGES_REQUESTED** [review]

File: paper.tex (hunk 3 above)
Comment (ivan-egorov42) [reply paper.tex]: @Graur OK, I understood.

**Comment by ivan-egorov42** [comment]
@Graur Fixed it. Is it better now?

File: paper.tex [hunk 4]
```diff
@@ -427,14 +427,16 @@ \subsection{Classes}
...
 
 \begin{ffcode}
++alias org.eolang.cage
+
 [] > book
   [i] > new
     cage 0 > id
     seq > @
       id.write i
       []
```
Comment (Graur) [inline paper.tex:435]: @ivan-egorov42 Does it compile?  Can you check it by `book.new (1) > @` ?

**Review by Graur — CHANGES_REQUESTED** [review]
@ivan-egorov42 Much better now, thanks! Just one clarification:

File: paper.tex [hunk 5]
```diff
@@ -427,14 +427,16 @@ \subsection{Classes}
 It can be mapped to the following EO code, where a class becomes a factory of objects:
 
 \begin{ffcode}
++alias org.eolang.cage
```
Comment (Graur) [inline paper.tex]: @ivan-egorov42 I think we don't need this alias. Can you check it?

**Comment by ivan-egorov42** [comment]
@Graur yes, you're right, it compiles without alias. Also, I fixed some mistakes. Here is the test for code - #6. So it works.

**Review by Graur — APPROVED** [review]
@ivan-egorov42 Thanks!
@yegor256 Please, have a look

**Comment by yegor256** [comment]
@ivan-egorov42 thanks!

**Pull request merged**